packages/scripts/src/generate-readme/generator.ts

```diff
@@ -128,8 +128,10 @@ export class ReadmeGenerator {
       )
 
       const adapter = adapterImport.adapter as Adapter
-      const adapterSettings = (adapter.config as unknown as { definition: SettingsDefinitionMap })
-        .definition
+      console.log(adapter)
```

Comment:
Was this added on purpose? If it was maybe we can append a message so anyone debugging would know what we're printing or why